api/server/server.go

```diff
@@ -658,10 +658,6 @@ func (s *Server) postCommit(version version.Version, w http.ResponseWriter, r *h
 		return err
 	}
 
-	if c == nil {
-		c = &runconfig.Config{}
-	}
-
```

Comment:
I removed this from api controller cause I think it's more appropriate to check this an in case instantiate a new Config directly in Commit routine